app/assets/javascripts/dashboard/explorative_annotations_view.js

```diff
@@ -336,6 +336,30 @@ class ExplorativeAnnotationsView extends React.PureComponent<Props, State> {
     );
   }
 
+  renderNameWithDescription(tracing: APIAnnotationType) {
+    return (
+      <Popover
+        title="Description"
+        trigger="hover"
+        content={
+          <div style={{ maxWidth: 400 }}>
+            {tracing.description && tracing.description !== ""
+              ? tracing.description
+              : "<no description>"}
+          </div>
+        }
+        placement="topLeft"
+      >
+        <div>
+          <EditableTextLabel
```

Comment:
Works well! However, I find it a bit weird from a UX perspective, that the popover always shows when you hover over the edit button or the name input. I'd change the structure as follows:

```
<div>
  <EditableTextLabel ... />
  <Popover>
    <i className="fa fa-align-justify" />
  </Popover>
</div>
```

That way, the description is a separate thing.